build/assets/file_type_cheader.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="16"
+   height="16"
+   viewBox="0 0 16 16"
+   version="1.1"
+   id="svg7"
+   sodipodi:docname="file_type_cheader.svg"
+   inkscape:version="1.2.2 (b0a8486541, 2022-12-01)"
```

Comment:
can Inkscape maybe export a clean svg without all this useless (for us) metadata in it?

Reply:
Okay I searched and found there is `optimized svg`, which removed all those misc information.